scripts/bsd/netbsd-set-cron.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+# Set cron job to get and run startup scripts
+crontab -r
+(crontab -l 2>/dev/null; echo '@reboot (/usr/pkg/bin/curl -s -L "http://metadata.google.internal/computeMetadata/v1/instance/attributes/startup-script" -H "Metadata-Flavor: Google") | /bin/sh') | crontab -
```

Comment:
Why is this done via cron? There should at least be a comment explaining this...